LICENSE

```diff
@@ -1 +1,5 @@
-TODO: Add your license here.
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
Can you remove this license? I'll need to update this to match the licenses in the actual source (whatever the Flutter project uses).

Reply:
I have removed it.